SPECS/rust/rust.spec

```diff
@@ -164,14 +165,25 @@ rm %{buildroot}%{_bindir}/*.old
 
 %files doc
 %license LICENSE-APACHE LICENSE-MIT LICENSE-THIRD-PARTY COPYRIGHT
-%doc %{_docdir}/%{name}/html/*
```

Comment:
I am seeing this when building, at least in my environment: WARN: (rust-doc-1.85.0-1.azl3.noarch.rpm) has license warning

Reply:
Discussed to revisit internally. The rust-doc package has a lot of file that are being flagged as bad doc files ....